biogrid/apps/biogrid-mvp/src/app/components/simulation-board.tsx

```diff
@@ -127,46 +127,51 @@ export const SimulationBoard = (props: SimBoardProps) => {
       ))}
       <div className="grid-line-wrapper">
         <svg width={boardWidth} height={boardHeight}>
-          {props.lines.map((line) => (
-            <>
-              <defs>
-                <marker
-                  id="arrowhead"
-                  markerWidth="10"
-                  markerHeight="7"
-                  refX="0"
-                  refY="3.5"
-                  orient="auto"
-                >
-                  <polygon points="0 0, 10 3.5, 0 7" />
-                </marker>
-              </defs>
-              <line
-                // Add half of the icon width in order to center the lines
-                x1={`${kilometersToCSSWidth(
-                  itemsByName[line.fromItem].x,
-                  SIZES.ICON_WIDTH_PERCENT / 2
-                )}`}
-                y1={`${kilometersToCSSHeight(
-                  itemsByName[line.fromItem].y,
-                  SIZES.ICON_WIDTH_PERCENT / 2
-                )}`}
-                x2={`${kilometersToCSSWidth(
-                  itemsByName[line.toItem].x,
-                  SIZES.ICON_WIDTH_PERCENT / 2
-                )}`}
-                y2={`${kilometersToCSSHeight(
-                  itemsByName[line.toItem].y,
-                  SIZES.ICON_WIDTH_PERCENT / 2
-                )}`}
-                marker-end="url(#arrowhead)"
-                stroke={(line.powerThroughLinesKiloWatts || 0) > 0 ? 'blue' : 'grey'}
-                stroke-width={kiloWattToThickness(
-                  line.powerThroughLinesKiloWatts || 1
-                )}
-              />
-            </>
-          ))}
+          {props.lines.map((line) => {
+            const lineThickness = kiloWattToThickness(
+              line.powerThroughLinesKiloWatts || 1
```

Comment:
is it possible that powerThroughLinesKiloWatts is ever set to 0 on purpose?

Reply:
I do not believe so. The edges only exist if items are connected